### PR TITLE
meta update

### PR DESCRIPTION
**Pull request by 532910**

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -1,7 +1,4 @@
 ---
-# Copyright (C) 2020 NSPCC <https://nspcc.ru/>
-# SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (roman-khimov) [inline meta/main.yml:3]: Why?

File: meta/main.yml [hunk 2]
```diff
@@ -13,10 +10,8 @@ galaxy_info:
 
   platforms:
     - name: Debian
-      versions:
-        - buster
+
```
Comment (roman-khimov) [inline meta/main.yml:13]: Why?

Comment (532910) [reply meta/main.yml:13]: 1. it's unmaintained value
2. did this record mean bullseye or bookworm were not supported?

File: meta/main.yml (hunk 1 above)
Comment (532910) [reply meta/main.yml:3]: just a garbage

the license is specified on the line 8 and in the LICENSE file

Comment (roman-khimov) [reply meta/main.yml:3]: SPDX makes things easier to process by various tools, removing it doesn't help anyone.

File: meta/main.yml (hunk 2 above)
Comment (roman-khimov) [reply meta/main.yml:13]: Do we support any version now? Are you sure it works for any of them? Why there is nothing written about it in the commit message?

**Pull request merged**